mozilla/donate-wagtail
# Bumped Django and AB Testing

Partially Closes #1457 

**Note**: There was a missing migration. I ran migrations and plopped it in this branch to pass CI. It was a small migration. 
**Note:** We also can't upgrade to Wagtail 2.12 yet because we have packages that specify wagtail<2.12. Those will need version bumps because we can upgrade to Wagtail 2.12 and 2.13. 

## Testing steps 
1. Checkout this branch 
2. run `inv new-env`. No problems should arise. 
3. Poke around on the localhost website to ensure things are working smoothly.
